qcsubmit/workflow_components/filters.py

```diff
@@ -134,6 +138,10 @@ class ElementFilter(BasicSettings, CustomWorkflowComponent):
         "I",
     ]
 
+    cache: Union[List[int], None] = None
```

Comment:
Lets make this a private parameter in the base class so all components can use it and it won't make its way into the json that way. We can also make the base class `apply_finalize` function clear the cache.

Reply:
Done!